script.js

```diff
@@ -0,0 +1,70 @@
+const navBar = document.getElementById('navbar');
+const navIcon = document.getElementById('nav-icon');
+const navMenuLinks = document.getElementById('nav-menu-links');
+const body = document.querySelector('body');
+const links = document.querySelectorAll('.nav_menu__container a');
+
+window.addEventListener('scroll', () => {
+  if (window.scrollY > 1) navBar.style.boxShadow = '2px 2px 2px 1px rgba(0, 0, 0, 0.2)';
+  else navBar.style.boxShadow = 'none';
+});
+
+function handler() {
+  if (window.innerWidth > 820) return;
+  if (navIcon.firstElementChild.id === 'hamburger-icon') {
+    body.style.overflowY = 'hidden';
+    navIcon.firstElementChild.src = './img/close-icon.svg';
+    navIcon.firstElementChild.id = 'close-icon';
+  } else {
+    body.style.overflowY = 'scroll';
+    navIcon.firstElementChild.src = './img/hamburger-icon.svg';
+    navIcon.firstElementChild.id = 'hamburger-icon';
+  }
+
+  navMenuLinks.classList.toggle('links-on-clicked');
+  navBar.classList.toggle('nav-on-clicked');
+  body.classList.toggle('dropout-filler');
+}
+
+links.forEach((link) => {
+  link.addEventListener('click', handler);
+});
+
+navIcon.addEventListener('click', handler);
+
+window.addEventListener('resize', () => {
+  links.forEach((link) => {
+    link.removeEventListener('click', handler);
+  });
+
+  links.forEach((link) => {
+    link.addEventListener('click', handler);
+  });
+
+  navMenuLinks.classList.remove('links-on-clicked');
+  navBar.classList.remove('nav-on-clicked');
+  body.classList.remove('dropout-filler');
+  navIcon.firstElementChild.src = './img/hamburger-icon.svg';
+  navIcon.firstElementChild.id = 'hamburger-icon';
+  body.style.overflowY = 'scroll';
+});
+
+const emailValue = document.querySelector('.email-value');
+
+const submitButton = document.querySelector('.button-email');
+
+const validationMessage = document.querySelector('.error');
+
+submitButton.addEventListener('submit', (event) => {
+  const emailValueContents = emailValue.value;
+  function UpperCaseChecker() {
+    const upperCaseCharaters = /[ABCDEFGHIJKLMNOPQRSTUVWXYZ]/;
+    return upperCaseCharaters.test(emailValueContents);
+  }
+
+  if (UpperCaseChecker() === true) {
```

Comment:
[Optional] Good job! I just wanted to provide a hint for simplifying the code. So what do you think about transforming this code into one line? I mean Something like this 👇
```javascript
if(emailValue.value !== emailValue.value.toLowerCase()) {
 // The validation error here
}

```